src/Orchard/Data/Migration/Interpreters/MySqlCommandInterpreter.cs

```diff
@@ -82,5 +84,31 @@ public class MySqlCommandInterpreter : ICommandInterpreter<AlterColumnCommand> {
                 return tableName;
             return _shellSettings.DataTablePrefix + "_" + tableName;
         }
+
+        public string[] CreateStatements(AddIndexCommand command) {
```

Comment:
Put a static prefix length value. 
Also reference the actual bugs  #4780 and #4779? and #5366